### PR TITLE
Community site updates 

### PR DESCRIPTION
**Pull request by skotzko**
Made some small updates / cleanup to community site prep for 1.0 launch. Typesafe will be adding callout on JVM Akka site that directs to Akka .NET site in reciprocation.

- Add "We are reactive" banner sized so it doesn't block nav items as it did before
- Add callout to JVM partner project in homepage project description
- HTML readability / formatting
- Remove community port warning from team page (redundant vs homepage/footer)

### CODE REVIEW

**Comment by skotzko** [comment]
@rogeralsing community site changes I mentioned to you before. Sorry for delay. If looks good to you let's get it up there so that Typesafe reciprocates on JVM Akka site before the 1.0 launch.

**Comment by rogeralsing** [comment]
I had to revert the Team page, it was bonkered beyond saving with all those extra `</div>`'s 
Can you make a new PR with only the relevant changes?

File: pages/team.html [hunk 1]
```diff
@@ -1,266 +1,236 @@
...
+        <a href="http://twitter.com/thinkb4coding" class="btn ico-only btn-twitter btn-xs"><i class="fa fa-twitter"></i></a>
+        <a href="https://github.com/thinkbeforecoding" class="btn ico-only btn-stackoverflow btn-xs"><i class="fa fa-github"></i></a>
+    </div>
+    </div>
+    <p class="noborder nomargin nopadding">Jérémie is an F# developer hailing from Paris, France. Follow him on <a href="https://twitter.com/thinkb4coding">Twitter</a> or at his personal blog, <a href="http://thinkbeforecoding.com/">Think Before Coding</a>.</p>
+    </div>
+    </div>
+    </div>
+    </div>
+    </div>
```
Comment (rogeralsing) [inline pages/team.html]: There are a lot of extra end divs here for some reason?

**Comment by rogeralsing** [comment]
align text

That "learn" looks somewhat alone over there, add linebreak?

**Comment by skotzko** [comment]
@rogeralsing thanks. I'll add fixes and submit a new PR

**Comment by rogeralsing** [comment]
I did some manual edits yesterday to fix the broken bits. Check the live site, everything ok there?

**Comment by skotzko** [comment]
@rogeralsing I'd just add 20px margin top to the "Learn more about Akka for the JVM here" <p>

**Pull request merged**